src/trpc/server/initTRPC.ts

```diff
@@ -1,31 +1,24 @@
-import {
-  createRouterWithContext,
-  createNewContext,
-  pipedResolver,
-  zod,
-  mergeRouters,
-  error,
-} from '.';
+import { createRouterWithContext, mergeRouters } from './router';
+import { createBuilder as createProcedure } from './procedure';
+import { createMiddlewareFactory } from './middleware';
 
 export function initTRPC<TContext>() {
   return {
     /**
-     * Create procedure resolver
+     * Builder object for creating procedures
      */
-    resolver: pipedResolver<TContext>(),
+    procedure: createProcedure<TContext>(),
```

Comment:
Should this be called `builder` or something else perhaps?

Reply:
I think `procedure` is fine, maybe `procedureBuilder` if you want to be more clear about what it is.

Reply:
Could `.from` make sense? Like you’re building a procedure from another [partial] procedure [builder]?